evdev/src/Evdev/Codes.chs

```diff
@@ -581,6 +613,57 @@ module Evdev.Codes where
     BTN_TRIGGER_HAPPY40 as BtnTriggerHappy40}
     deriving (Bounded, Eq, Ord, Read, Show) #}
 
+pattern KeyHanguel :: Key
+pattern KeyHanguel =  KeyHangeul
```

Comment:
```suggestion
pattern KeyHanguel = KeyHangeul
```

I don't generally like this sort of alignment. It isn't consistent here anyway.

Reply:
Sure.